server/src/events.mjs

```diff
@@ -92,10 +93,117 @@ function sendEventListenerAck(ws, oMsg) {
   logger.debug(`Sent event listener ack message: ${ackMessage}`);
 }
 
+// sendEvent to handle post API event calls, including pre- and post- event trigger processing
+function sendEvent(ws, method, result, msg, fSuccess, fErr, fFatalErr) {
+  try {
+    if ( !isRegisteredEventListener(method) ) {
+      fErr.call(null, method);
+
+    } else {
+       // Fire pre trigger if there is one for this method
+       if ( method in eventTriggers ) {
+        if ( 'pre' in eventTriggers[method] ) {
+          try {
+            const ctx = {
+              logger: logger,
+              setTimeout: setTimeout,
+              setInterval: setInterval,
+              sendEvent: function(method, result, msg) {
+                function fSuccess() {logger.info({status: 'SUCCESS'})}
+                function fErr(method) {
+                  logger.info({status: 'ERROR',
```

Comment:
The logger methods take strings, not objects.

Reply:
modifed this logger statements